docs/src/maths/propagators.md

```diff
@@ -47,21 +47,21 @@ This document will explain this point of view in some detail.
 
 ##### Some terminology/conventions
 
-Let ``p`` be an element of type M, which is defined by some assignment of numbers ``x_1,...,x_m``,
-say ``(x_1,...,x_m) = (a_1,...,1_m)``
+Let ``p`` be an element of type ``M``, which is defined by some assignment of numbers ``x_1, \dots, x_m``,
+say ``(x_1, \dots, x_m) = (a_1, \dots, a_m)``
 
-A _function_ ``f:M \to K`` on ``M`` is (for simplicity) a polynomial ``K[x_1, ... x_m]``
+A _function_ ``f:M \to K`` on ``M`` is (for simplicity) a polynomial ``K[x_1, \dots, x_m]``
```

Comment:
I don't fully understand the concept of this section. `f in K[x_1, ... x_m]` is defined here, but is not used in the later section. (Same for the above `(a_1,...,1_m)`.)

I feel the entire section could be improved, but I just fixed some typos in this PR.